librsvg.yaml

```diff
@@ -0,0 +1,93 @@
+package:
+  name: librsvg
+  version: 2.57.1
+  epoch: 0
+  description: SAX-based renderer for SVG files into a GdkPixbuf
+  copyright:
+    - license: LGPL-2.1-or-later
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - bzip2-dev
+      - ca-certificates-bundle
+      - cairo-dev
+      - fribidi-dev
+      - gi-docgen
+      - glib-dev
+      - gobject-introspection-dev
+      - graphviz
+      - graphviz-dev
+      - gtk-3-dev
+      - harfbuzz-dev
+      - libgsf-dev
+      - libxft-dev
+      - libxml2-dev
+      - pango-dev
+      - pkgconf
+      - pkgconf-dev
+      - py3-docutils
+      - py3-gi-docgen
+      - rust
+      - smartypants
+      - ttf-dejavu
+      - vala
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://gitlab.gnome.org/GNOME/librsvg.git
+      expected-commit: 34cc425b40bb6299868157fe98c945c30bbc3b71
+      tag: ${{vars.mangled-package-version}}
+
+  - runs: |
+      NOCONFIGURE=1 ./autogen.sh
+      dot -c
+      ./configure \
+        --build=$CBUILD \
+        --host=$CHOST \
+        --prefix=/usr \
+        --libexecdir=/usr/lib/${{package.name}} \
+        --disable-static \
+        --enable-vala
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
+  - name: librsvg-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - librsvg
+        - rsvg-convert
+    description: librsvg dev
+
+  - name: librsvg-doc
+    pipeline:
+      - uses: split/manpages
+    description: librsvg manpages
+
+  - name: rsvg-convert
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/bin
+          mv "${{targets.destdir}}"/usr/bin/rsvg-convert "${{targets.subpkgdir}}"/usr/bin
+
+  - name: rsvg-convert-doc
+    pipeline:
+      - uses: split/manpages
+    description: librsvg manpages
+
+# we use gitlab for this package, so we need to disable the release monitor otherwise CI fails
+update:
+  enabled: false
```

Comment:
We can still use release monitoring https://release-monitoring.org/project/5420/